python/mxnet/gluon/contrib/estimator/estimator.py

```diff
@@ -59,6 +59,8 @@ class Estimator(object):
         Trainer to apply optimizer on network parameters.
     context : Context or list of Context
         Device(s) to run the training on.
+    evaluation_loss: gluon.loss.loss
+        Loss (objective) function to calculate during evaluation.
```

Comment:
It's not clear that `evaluation_loss` will use `loss` if it is None. Could you improve the description here?